bot/code_review_bot/workflow.py

```diff
@@ -25,12 +25,16 @@
 from code_review_bot.tasks.default import DefaultTask
 from code_review_bot.tasks.infer import InferTask
 from code_review_bot.tasks.lint import MozLintTask
+from code_review_bot.tasks.upload import DocUploadTask
 
 logger = structlog.get_logger(__name__)
 
 TASKCLUSTER_NAMESPACE = "project.relman.{channel}.code-review.{name}"
 TASKCLUSTER_INDEX_TTL = 7  # in days
 
+COMMENT_LINK_TO_DOC = """
+We think you might have touched the doc files, generated doc can be accessed [here]({link_to_doc}).
```

Comment:
I'd write something like:
```suggestion
I think you might have touched the documentation, so I rendered it for you [here]({link_to_doc}).
```

Reply:
Or:
```suggestion
You have touched the documentation, you can find it rendered [here]({link_to_doc}).
```

Reply:
please take in account Marco's comment.
It could be:
You have touched the documentation, you can find it rendered [here]({link_to_doc}) for a week.

Reply:
Don't forget this!

Reply:
you forgot the "for a week" in the commit message :)
it is important to highlight that, these urls will expire